wp-content/themes/redmond/web/scss/modules/_icon-columns.scss

```diff
@@ -1 +1,35 @@
-// icons below our services section in home page
\ No newline at end of file
+.icon-columns {
+  margin-bottom: rem-calc(70px);
+  li a {
+    text-transform: uppercase;
+    color: $secondary-dark;
+  }
+
+  li a:hover {
+    color: $primary;
+  }
+  .icon-title {
+    margin-bottom: rem-calc(30px);
+    img {
+      float: left;
+      margin-right: rem-calc(10px);
+    }
+    h2 {
+      text-transform: uppercase;
+    }
+    h2 small {
```

Comment:
ignore this. in another pr you can more .icon-title and the h2 logic into app.scss